Makefile

```diff
@@ -442,7 +442,7 @@ e2e-deploy-base-ratify: e2e-notation-setup e2e-notation-leaf-cert-setup e2e-inli
 	--set oras.useHttp=true \
 	--set cosign.enabled=false \
 	--set-file dockerConfig="mount_config.json" \
-	--set logLevel=debug
+	--set logger.level=debug
```

Comment:
just an fyi that this is a breaking change in the helm chart. we need to add this in the release notes